src/client/sound/ogg.c

```diff
@@ -709,3 +710,62 @@ OGG_Shutdown(void)
 
 	ogg_started = false;
 }
+
+void
+OGG_LoadAsWav(char *filename, wavinfo_t *info, void **buffer)
```

Comment:
I know lots of code in q2 does this, but maybe we should finally start using `const char*` for strings, at least in new code

Reply:
Filename has used in `FS_LoadFile` and `FS_LoadFile` has used `char*`, i have updated chars to const where it not required to add `const char*` -> `char*` before use.